app/app.js

```diff
@@ -1,27 +1,23 @@
-var router = require( './app.router' );
-require( 'script!jquery' );
-require( 'script!jquery_validation' );
-require( 'script!validation_additional_methods' );
-require( 'script!sammy' );
-require( 'script!mustache' );
+var router = require("./app.router");
+require("script!jquery");
+require("script!jquery_validation");
+require("script!validation_additional_methods");
+require("script!sammy");
+require("script!mustache");
```

Comment:
Have you run eslint? It should has single quotes and spaces inside the parentesis.